assignments/array-methods.js

```diff
@@ -55,29 +55,79 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+let fullName = runners.map((arr) => {
+    return arr.first_name + ' ' + arr.last_name;
+});
 console.log(fullName);
```

Comment:
In the solution code, they have a forEach here, but this is awesome, too, very nice!